README.md

```diff
@@ -16,6 +16,9 @@ The current focus is on nonlinear least squares with support for sparsity, batch
 - Prerequisites
     - We *strongly* recommend you install `theseus` in a venv or conda environment.
     - Theseus requires `torch` installation. To install for your particular CPU/CUDA configuration, follow the instructions in the PyTorch [website](https://pytorch.org/get-started/locally/).
+    - Theseus also requires [`suitesparse`](https://people.engr.tamu.edu/davis/suitesparse.html), which you can install via:
+        - `sudo apt-get install libsuitesparse-dev` (Ubuntu).
+        - `conda install -c conda-forge suitesparse` (Mac).
```

Comment:
Sorry for the late review, but since you're strongly recommending users to install theseus in a conda environment, shouldn't `conda install` be the go-to method for both Ubuntu and Mac?

Reply:
I actually know several people (me included), who mainly uses `conda` to create a new virtual env, and after exclusively uses `pip install` for installing packages. On the other hand, I think it makes sense to change "strongly recommend to install ... a conda env" for strongly recommending to create a new Python env, either via conda, or via `venv`.